osbs/api.py

```diff
@@ -614,6 +614,7 @@ def _do_create_prod_build(self, git_uri, git_ref,
                               reactor_config_override=None,
                               parent_images_digests=None,
                               git_commit_depth=None,
+                              allowed_platform=None,
```

Comment:
I can't think of a better name for this :-(

Reply:
maybe `operator_uploader` or `operator_uploader_platform`?

Reply:
The best I came up with was `noarch_platform`, i.e. the platform whose worker can perform operations for which it doesn't matter which platform is used. But the fact that I have to explain it probably means it's a bad name.

I prefer either of `operator_uploader` or `operator_uploader_platform` to `allowed_platform`.

Reply:
Or even `any_platform`.

Reply:
I vote for `operator_uploader_platform`, that names describes what is that param used for

Reply:
BTW, in design doc we agreed on `operator_manifests_extract_platform`